### ISSUE TITLE
the shape of rel_matrix

### ISSUE THREAD

**Issue by fyubang** [laloc2496/bert-jointly-relation-entity-extraciton#1]
Thanks for your contributions. It is amazing. I have a question about the rel_matrix shape
```
    def convert_rel_ids(self, index_rels, label_rels):
        num_rels = len(self.rel_list)
        rel_matrix = torch.zeros(1, self.max_len, self.rel_max_len * num_rels)
        for i, (idxs, labels) in enumerate(zip(index_rels, label_rels)):
            for idx, label in zip(idxs, labels):
                try:
                    rel_matrix[0, i, idx * num_rels + self.rel_list.index(label)] = 1
                except IndexError:
                    print(rel_matrix.size())
                    print(i, idx, num_rels, self.rel_list.index(label))
                    print(0, i, idx * num_rels + self.rel_list.index(label))
                    raise ValueError('out of index range')
        return rel_matrix
```
why the shape[-1] of rel_matrix is `self.rel_max_len * num_rels`? I think it should be `self.max_len*num_rels` .
Otherwise` idx * num_rels + self.rel_list.index(label)` will be out of range.

**Comment by lipingtang17** [comment]
I agree with you @fyubang
